### PR TITLE
Fix various typos in docs, user-facing code, and source comments

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,./lib/libUPnP/Neptune -L bloaded,busses,inout,lod,medias,parm`

### CODE REVIEW

**Review by garbear — APPROVED** [review]
Spelling fixes look correct. You even caught comments with grammar problems and no spelling problems.

**Review by ksooo — COMMENTED** [review]
Changes in xbmc/pvr look good to me. Thanks.

**Review by DaveTBlake — COMMENTED** [review]
Changes to files in xbmc/music look OK to me

File: cmake/treedata/windowsstore/subdirs.txt [hunk 1]
```diff
@@ -7,8 +7,8 @@ xbmc/platform/win10                    platform/win10
 xbmc/platform/win10/filesystem         platform/win10/filesystem
 xbmc/platform/win10/network            platform/win10/network
 xbmc/platform/win10/peripherals        platform/win10/peripherals
-xbmc/platform/win10/powermanagement    platfrom/win10/powermanagement
-xbmc/platform/win10/storage            platfrom/win10/storage
+xbmc/platform/win10/powermanagement    platform/win10/powermanagement
+xbmc/platform/win10/storage            platform/win10/storage
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply cmake/treedata/windowsstore/subdirs.txt:11]: Probably unbreaks something :wink:

File: addons/metadata.generic.albums/lib/musicbrainz.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def musicbrainz_albumfind(data, artist, album):
         if item.get('score'):
             releasescore = item['score'] / 100.0
             # if the release is in the releasegroup with most releases, it is considered the most accurate one
-            # (this also helps with prefering official releases over bootlegs, assuming there are more variations of an official release than of a bootleg)
+            # (this also helps with preferring official releases over bootlegs, assuming there are more variations of an official release than of a bootleg)
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply addons/metadata.generic.albums/lib/musicbrainz.py:68]: Submitted fixes to upstream repo. Ref: https://gitlab.com/ronie/metadata.generic.albums/-/merge_requests/1  
Should I revert these fixes here or just keep them and let them be overwritten in the next sync?

**Comment by luzpaz** [comment]
rebased

**Comment by howie-f** [comment]
whoops, sorry @luzpaz. function names changed. please rebase and this should be ready to go

**Comment by luzpaz** [comment]
Rebased and fixed merge conflict

**Review by howie-f — APPROVED** [review]
very nice and extensive cleanup 👍 .. thanks a lot. i‘m going to merge this later today

File: addons/webinterface.default/lib/video-js/video.js [hunk 3]
```diff
@@ -2264,7 +2264,7 @@ var Component = function () {
   /**
    * Clears a timeout that gets created via `window.setTimeout` or
    * {@link Component#setTimeout}. If you set a timeout via {@link Component#setTimeout}
-   * use this function instead of `window.clearTimout`. If you don't your dispose
+   * use this function instead of `window.clearTimeout`. If you don't your dispose
```
Comment (enen92) [inline addons/webinterface.default/lib/video-js/video.js:2267]: These changes should have gone also upstream to chorus2

Comment (luzpaz) [reply addons/webinterface.default/lib/video-js/video.js:2267]: what is the repo address for upstream ?

Comment (enen92) [reply addons/webinterface.default/lib/video-js/video.js:2267]: https://github.com/xbmc/chorus2

Comment (howie-f) [reply addons/webinterface.default/lib/video-js/video.js:2267]: yes, missed it
https://github.com/xbmc/chorus2/blob/43750e50663ac75f671779ed998a279f31ff6023/dist/lib/video-js/video.js#L2267

Comment (luzpaz) [reply addons/webinterface.default/lib/video-js/video.js:2267]: Opened PR in chorus: https://github.com/xbmc/chorus2/pull/451

**Pull request merged**